.commitlintrc.json

```
@@ -15,6 +15,7 @@
         "core",
         "csi",
         "docs",
+        "external-script",
```

how about this for any external cluster change?
```suggestion
        "external",
```